core/domain/exp_jobs_one_off_test.py

```diff
@@ -92,6 +92,249 @@ def run_job_for_deleted_exp(
+                title='title',
+                objective='objective',
+                language_code='en',
+                tags=[],
+                blurb='',
+                author_notes='',
+                init_state_name=feconf.DEFAULT_INIT_STATE_NAME,
+                param_specs={},
+                param_changes=[],
+                skill_tags=['tag1', 'tag2'],
+            )
+            rights_manager.create_new_exploration_rights('exp_id1', 'uid_1')
+            commit_message = 'New exploration created with title \'title\'.'
+            exp_model1.commit('uid_1', commit_message, [{
+                'cmd': 'create_new',
+                'title': 'title',
+                'category': 'category',
+            }])
+
+            # exp_model1 = (
+            #     exp_models.ExplorationModel.get_by_id('exp_id1'))
+
+            self.assertIsNotNone(exp_model1.skill_tags)
+
+            self.assertIn('skill_tags', exp_model1._values) # pylint: disable=protected-access
+            self.assertIn('skill_tags', exp_model1._properties) # pylint: disable=protected-access
+
+            output = self._run_one_off_job()
+            self.assertItemsEqual(
+                [['SUCCESS_REMOVED - ExplorationModel', 1]], output)
+
+            migrated_exp_model1 = (
+                exp_models.ExplorationModel.get_by_id('exp_id1'))
+
+            self.assertNotIn(
+                'skill_tags', migrated_exp_model1._values)  # pylint: disable=protected-access
+            self.assertNotIn(
+                'skill_tags', migrated_exp_model1._properties)  # pylint: disable=protected-access
+
+            # Run job twice.
+            output = self._run_one_off_job()
+            self.assertItemsEqual(
+                [['SUCCESS_ALREADY_REMOVED - ExplorationModel', 1]], output)
+
+    def test_one_exp_models_with_deprecated_field_default_skin(self):
+        with self.swap(
+            exp_models,
+            'ExplorationModel',
+            MockExplorationModelWithDeprecatedFields
+        ):
+            exp_model2 = exp_models.ExplorationModel(
+                id='exp_id2',
+                category='category',
+                title='title',
+                objective='objective',
+                language_code='en',
+                tags=[],
+                blurb='',
+                author_notes='',
+                init_state_name=feconf.DEFAULT_INIT_STATE_NAME,
+                param_specs={},
+                param_changes=[],
+                default_skin='conversation_v1',
+            )
+            rights_manager.create_new_exploration_rights('exp_id2', 'uid_2')
+            commit_message = 'New exploration created with title \'title\'.'
+            exp_model2.commit('uid_2', commit_message, [{
+                'cmd': 'create_new',
+                'title': 'title',
+                'category': 'category',
+            }])
+
+            # exp_model2 = (
+            #     exp_models.ExplorationModel.get_by_id('exp_id2'))
```

Comment:
Do not keep commented code, either keep it or remove it completely. Ditto in other places.

Reply:
Done